admin/database/postgres/postgres.go

```diff
@@ -962,6 +962,66 @@ func (c *connection) DeleteExpiredDeploymentAuthTokens(ctx context.Context, rete
 	return parseErr("deployment auth token", err)
 }
 
+func (c *connection) FindMagicAuthTokens(ctx context.Context, projectID string, createdByUserID *string, afterID string, limit int) ([]*database.MagicAuthToken, error) {
+	var res []*magicAuthTokenDTO
+	if createdByUserID == nil {
+		err := c.getDB(ctx).SelectContext(ctx, &res, "SELECT t.* FROM magic_auth_tokens t WHERE t.project_id=$1 AND t.id>$2 ORDER BY t.id", projectID, afterID)
```

Comment:
makes sense because empty string is not a valid UUID so may be split it in two cases - if its empty string then no comparison just have order by and if not then have the comparison. Also I see limit is not being used.

Reply:
Fixed